doc/rst/language/spec/statements.rst

```diff
@@ -803,18 +803,19 @@ the following:
 
  * a top-level module name
  * a submodule of the current module
- * a module name currently in scope due to another ``use`` statement
+ * a module name currently in scope due to another ``use`` or ``import``
+   statement
  * any number of ``super`` components to indicate a number of parents of
    the current module (e.g. ``super.super.SomeModule``)
  * ``this`` to indicate the requested module is a submodule of the
    current module
 
-The names that are imported by a use statement are inserted in to a new
-scope that immediately encloses the scope within which the statement
-appears. This implies that the position of the use statement within a
-scope has no effect on its behavior. If a scope includes multiple use
-statements then the imported names are inserted in to a common enclosing
-scope.
+The names that are imported by a use statement are inserted in to a new scope
```

Comment:
This line has an example showing the importance of styling import/use. It may be difficult to tell "import" here is not referring to the `import` statement. If you don't go the styling way, maybe words can be chosen more carefully here?

Reply:
This also should be resolved now